composer.json

```diff
@@ -11,7 +11,7 @@
     "phpunit/phpunit": "^5 || ^6 || ^7 || ^8 || ^9",
     "roave/security-advisories": "dev-latest",
     "roots/wordpress": ">=5.2",
-    "squizlabs/php_codesniffer": "^3.3",
+    "squizlabs/php_codesniffer": "3.7.1",
```

Comment:
```suggestion
    "squizlabs/php_codesniffer": "^3.7",
```

We tend to not use as fixed version numbers in `composer.json`, to allow for patch updates.

Reply:
@tofumatt just a note that version constraint is slightly different in composer compared to npm. ie. `^3.7` will not only update to patch version, but also minor version (ie. 3.8...). For patch only, we need `^3.7.0`. This is an info I found out while working on this very IB. :smile: 

However all the deps in our composer JSON are in the former 2 segment style and allows (unintentionally?) for minor version update as well so this is not an issue specific to this one. 

Just wanted to flag. 

cc @aaemnnosttv @eugene-manuilov